library/std/src/macros.rs

```diff
@@ -88,6 +94,12 @@ macro_rules! print {
 /// println!("format {} arguments", "some");
 /// ```
 #[macro_export]
+#[doc(alias = "echo")]
+#[doc(alias = "print")]
```

Comment:
I don't think it's a good idea to add this alias. They'll find the `print` macro which talks about the `println` one.